frontend/sw360-portlet/src/main/resources/META-INF/resources/html/projects/includes/attachmentUsages.jspf

```diff
@@ -38,12 +38,18 @@
             <core_rt:if test="${projectList.size() > 1 or (projectList.size() == 1 and not empty projectList.get(0).linkedReleases)}">
                 (<a href="#" id="expandAll" class="text-primary"><liferay-ui:message key="expand.all" /> </a>|
                 <a href="#" id="collapseAll" class="text-primary"> <liferay-ui:message key="collapse.all" /></a>)
+                <select class="mx-2" id="releaseFilter">
+                    <option value="" class="textlabel stackedLabel"> -- Release Filter -- </option>
+                    <option value="1" class="textlabel stackedLabel">With attachments</option>
+                    <option value="0" class="textlabel stackedLabel">Without attachments</option>
```

Comment:
Since `Release Filter`, `With attachments`, `Without attachments` are hardcoded values.
Kindly use language.properties for I18N

Reply:
done.